fivetran/tests/mock/resource_external_logging_test.go

```diff
@@ -0,0 +1,414 @@
+package mock
+
+import (
+    "net/http"
+    "testing"
+    "time"
+
+    "github.com/fivetran/go-fivetran/tests/mock"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var (
+    externalLoggingPostHandler   *mock.Handler
+    externalLoggingPatchHandler  *mock.Handler
+    externalLoggingTestHandler   *mock.Handler
+    externalLoggingDeleteHandler *mock.Handler
+    testExternalLoggingData      map[string]interface{}
+
+    externalLoggingMappingGetHandler    *mock.Handler
+    externalLoggingMappingPostHandler   *mock.Handler
+    externalLoggingMappingDeleteHandler *mock.Handler
+)
+
+const (
+    externalLoggingMappingResponse = `
+    {
+        "id":                 "log_id",
+        "group_id":           "group_id",
+        "service":            "azure_monitor_log",
+        "enabled":            false,
+        "config":{
+            "workspace_id":   "workspace_id",
+            "primary_key":    "primary_key",
+            "role_arn":       "role_arn",
+            "region":         "region",
+            "log_group_name": "log_group_name",
+            "sub_domain":     "sub_domain",
+            "enable_ssl":     "enable_ssl",
```

Comment:
shouldn't it be boolean?
Same for go-fivetran - it is a bug. It's won't work with real API.
```suggestion
            "enable_ssl":     true,
```

Reply:
public class SplunkLogCredentials implements ApiVerifiable {
    @ConfigField(type = FieldType.CONFIG_SECRED)
    public boolean enableSSL = true;